### ISSUE TITLE
The sunbird is stiff

### ISSUE THREAD

**Issue by DracoHero** [Alex-the-666/AlexsMobs#2124]
Minecraft 1.20.1 - Forge 47.3.7 - Alex's Mobs 1.22.9

https://github.com/user-attachments/assets/6383c917-f82d-4718-b2a0-9c23def3a392

**Comment by SaltSouls** [comment]
Yeah I am also getting this issue. I've even gotten as many as three sunbirds spawning within close proximity to each other, all of which are just hovering there.

Edit: I would like that this is likely due to their pathfinding breaking either because they were spawned when out of a loaded chunk, or because they are trying to pathfind to a unloaded chunk. I'm not entirely sure. All I know is that it seems like their pathfinding AI just fails to work frequently.

**Comment by Vinzound** [comment]
I confirm, I also have that problem with sunbirds and occasionally with hummingbirds as well :(

**Comment by Vinzound** [comment]
These two guys have been in the same place since I started playing this seed.
![Image](https://github.com/user-attachments/assets/12bb3529-fd3a-4cdc-a10b-33753bd5a1ad)
